src/styles.scss

```diff
@@ -706,6 +706,22 @@ mat-card-footer {
   margin-top: 1em;
 }
 
+.emoji-mart-scroll {
+  height: 205px !important;
+  padding: 0 !important;
+}
+
+.emoji-mart-emoji:hover:before {
+  top: 2px !important;
+  left: 4px !important;
+}
+
+.picker {
+  display: block;
+  position: fixed;
```

Comment:
Did you test this with scrolling? Couldn't get the picker to render properly when replying to comments. Will test this PR and see.

Reply:
I moved this out again, can't use it on global.